source/speech/speech.py

```diff
@@ -369,6 +390,8 @@ def _getSpellingSpeechWithoutCharMode(
 		capPitchChange: int,
 		beepForCapitals: bool,
 		fallbackToCharIfNoDescription: bool = True,
+		unicodeNormalization: bool = False,
+		reportNormalizedForCharacterNavigation: bool = False,

```

Comment:
please add docstring entries for new parameters

Reply:
I'll update the doc string.